### ISSUE TITLE
Bug: Y Axis Controls / Min = 0 sometimes ignored

### ISSUE THREAD

**Issue by core-process** [influxdata/chronograf#1857]
Thanks for the Display Options Tab and for the Y Axis Controls. Lovin it! :-)

OS: Ubuntu 16.10, 64bit
Version of Chronograf: 1.3.6-0 (Docker Image)
Version of InfluxDB: 1.3.2 (Docker Image)
Version of Kapacitor: 1.3.2 (Docker Image)

I observed the following bug: Chronograf ignores the "Min = 0" setting now and then, when selecting a smaller time frame (e.g. 15 min) on specific time series.

My settings:
![screenshot at 2017-08-10 17-05-51](https://user-images.githubusercontent.com/1226918/29178080-cf1e67e4-7df0-11e7-8e96-6f7186324c84.png)

This is OK:
![screenshot at 2017-08-10 17-06-37](https://user-images.githubusercontent.com/1226918/29178108-e269a796-7df0-11e7-880f-9a779271c72f.png)

This is broken:
![screenshot at 2017-08-10 17-07-04](https://user-images.githubusercontent.com/1226918/29178111-e9170fe8-7df0-11e7-9eae-e25b40d3b3f7.png)

**Comment by 121watts** [comment]
@core-process thanks for writing in!  It looks like the entire y-axis is jacked up.  It's going to be tough for me to investigate without knowing how the time series data looks.  This is just for smaller time ranges?

**Comment by core-process** [comment]
@121watts can you reproduce this issue with the following data? [export.json.zip](https://github.com/influxdata/chronograf/files/1216289/export.json.zip)

Yes, it is just for smaller time ranges. But sometimes it works with a timerange of 1h, sometimes not. So far it is always broken with a 15min time range.

**Comment by 121watts** [comment]
@core-process let me check it out!  thanks for exporting that

**Comment by 121watts** [comment]
@core-process I tried to recreate the issue using the dataset provided and could not reproduce.  Closing this for now :)

**Comment by core-process** [comment]
@121watts can we arrange a teamviewer session or something similar to figure out the essential parameters for reproduction of this issue?

**Comment by 121watts** [comment]
@core-process Fore sure.  Another thought, if it's possible, is to send me the connection info for your data source and the exact query you're running:  watts@influxdata.com

**Comment by core-process** [comment]
@121watts thanks for your reply. I will try to create a self-contained example within a VirtualBox VM.

**Comment by core-process** [comment]
This is what I did to reproduce this issue:

- setup VM in VirtualBox
- install Ubuntu 17.04 Netboot
- install Docker CE
- deploy image influxdb:1.3.3
- deploy image chronograf:1.3.6.1
- generate data for the last 24h with the value 358.84 using minute intervals
- create a new dashboard and a new cell (leaving default size)
- use "GROUP BY time(1m)", Line Type, Min = 0
- open the dashboard and navigate the data via "Past 5/15 minutes", "Past hour", etc...

More details:

I used the following script to prepare the ISO image of the Netboot setup: https://github.com/core-process/linux-unattended-installation/tree/master/ubuntu/17.04

I used this manual to install Docker: https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/

I used the following commands to deploy InfluxDB and Chronograf:

```sh
docker network create example
docker run -d --net=example -p 8086:8086 -v /data/influxdb:/var/lib/influxdb --name=influxdb influxdb:1.3.3
docker run -d --net=example -p 8888:8888 -e "INFLUXDB_URL=http://influxdb:8086" -v /data/chronograf:/var/lib/chronograf --name=chronograf chronograf:1.3.6.1
```

I used the following script to generate the data:

```sh
# create database
curl -i -XPOST http://localhost:8086/query --data-urlencode "q=CREATE DATABASE example"

# create datapoints for the last 24 hours
for i in `seq 0 1440`;
do
  curl -i -XPOST 'http://localhost:8086/write?db=example' --data-binary "example,variant=alpha value=358.84 `date --date "-${i}min" +%s%N`"
done

# create datapoints continuously
while true
do
  curl -i -XPOST 'http://localhost:8086/write?db=example' --data-binary "example,variant=alpha value=358.84 `date +%s%N`"
  sleep 60s
done
```

I used the following query for the dashboard cell:

```sql
SELECT mean("value") AS "mean_value" FROM "example"."autogen"."example" WHERE time > :dashboardTime: GROUP BY time(1m)
```

I used the following "Display Options":

- Visualization Type: Line
- Title: example.value
- Min: 0
- Max: (blank = auto)

I used the following browser environment:

- Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36
- Windows size: 1920 x 1080 (including system bars)
- Ubuntu Mate 17.04

**Comment by core-process** [comment]
@121watts here you will find a VirtualBox VM https://drive.google.com/open?id=0B6A70E731hjqbFZpQ3RNU2JHTzA

- Import appliance into VirtualBox.
- Make sure the bridge network adapter is connected to your active network adapter. Or reconfigure to use internal networking.
- Login as "root" using password "root".
- Get current IP address with "ifconfig".
- Run ``docker start influxdb chronograf``.
- Run ``./generate-data.sh``. Wait a few seconds...
- Open browser and navigate to http://ip-of-vm:8888/sources/0/dashboards/1

**Comment by 121watts** [comment]
Wow, @core-process thank you for all your work getting this environment setup.  I will investigate!

**Comment by 121watts** [comment]
@core-process just merged in a fix with #1930.  Now, user defined min and max's should be respected.

**Comment by core-process** [comment]
@121watts Awesome, thanks! :-)
